src/query/xasl_cache.c

```diff
@@ -2403,11 +2404,21 @@ xcache_check_recompilation_threshold (THREAD_ENTRY * thread_p, XASL_CACHE_ENTRY
 	}
       assert (!VFID_ISNULL (&cls_info_p->ci_hfid.vfid));
 
-      if (file_get_num_user_pages (thread_p, &cls_info_p->ci_hfid.vfid, &npages) != NO_ERROR)
+      if (!use_stat_estimation)
```

Comment:
remove the use_stat_estimation variable,
How about changing it like below?
```suggestion
      if (!prm_get_bool_value (PRM_ID_USE_STAT_ESTIMATION))
```